docs/natspec-format.rst

```diff
@@ -60,7 +56,6 @@ The following example shows a contract and a function using all available tags.
     /// @notice You can use this contract for only the most basic simulation
     /// @dev All function calls are currently implemented without side effects
     contract Tree {
-        /// @author Mary A. Botanist
```

Comment:
Did this not compile with autor?

Reply:
It did, but this PR also adds deprecation for the `author` field for functions, I figured we don't want a deprecated field in the example.